pytorch_pfn_extras/torch_version.py

```diff
@@ -0,0 +1,9 @@
+import torch
+from packaging import version
+
+
+torch_version = version.Version(torch.__version__.split("+")[0])
+
+
+def is_available(ver: str) -> bool:
```

Comment:
How about defining it as 

```suggestion
def requires(version: str, *, package='torch') -> bool:
```

and get version via `pkg_resources.get_distribution(package).version` so that this function can be used for family packages such as `torchvision` as well?